kie-osgi/kie-karaf-features/src/main/filtered-resources/repository/features-core.xml

```diff
@@ -28,7 +28,7 @@
 
   <feature name="drools-jpa" version="${project.version}" description="Drools JPA">
     <feature version="${project.version}">drools-module</feature>
-    <feature version="[4,5)">spring-orm</feature>
+    <feature version="[4,6)">spring-orm</feature>
```

Comment:
I used `[4,6)` rather than `[5,6]` because the default Karaf (used by CI) still uses Spring 4.